org.eclipse.xtext.tests/src/org/eclipse/xtext/xtext/Issue1161Test.xtend

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
```

Comment:
Lorenzo, the initial copyright notice is assigned to the provider of the initial API. Thanks for having itemis here, but actually it is yours. If you are committing in behalf of your employer, then add your companies' name. If you are doing this privately, add yourself.
Also do this for other added files, please.

Reply:
Ops, thanks!